kube-scheduler-healthcheck

```diff
@@ -18,7 +18,11 @@ endOfPodDef
 
 alias kubectl="timeout -t 30 kubectl --namespace \$TEST_POD_NAMESPACE"
  
-kubectl delete pod $TEST_POD_NAME || true
+kubectl delete pod $TEST_POD_NAME --grace-period=0 || true
+sleep 1
+kubectl delete pod --grace-period=0 --force || true
 kubectl create -f /tmp/pod.yaml
 until [ "$(kubectl get ${TEST_POD_NAME} -o jsonpath={.status.phase})" != "Pending" ]; do sleep 1; done
-kubectl delete pod $TEST_POD_NAME
+kubectl delete pod $TEST_POD_NAME --grace-period=0
+sleep 1
+kubectl delete pod --grace-period=0 --force
```

Comment:
@mikebryant wondering whether to `|| true` here as well?

Reply:
Yes, we'll need to